src/framework/core/js/Fluid.js

```diff
@@ -1607,7 +1786,7 @@ var fluid = fluid || fluid_2_0;
     fluid.makeComponents = function (components) {
```

Comment:
Can we either mark this as unsupported, document it, or move it to a testing utility?